mmedit/structures/edit_data_sample.py

```diff
@@ -202,6 +203,8 @@ def set_tensor_data(self, data: dict) -> None:
         for k, v in data.items():
             if k == 'gt_label':
                 self.set_gt_label(v)
+            elif k == 'prompt':
+                self.set_prompt(v)
```

Comment:
We don't need to define new methods (including `set_prompt`, etc.) for `prompt`. 
We can use `self.set_field(prompt, 'prompt')`